bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/FontMetrics.java

```diff
@@ -208,10 +214,11 @@ public int hashCode() {
  *
  * @noreference This method is not intended to be referenced by clients.
  */
-public static FontMetrics win32_new(TEXTMETRIC handle) {
+public static FontMetrics win32_new(TEXTMETRIC handle, int nativeZoom) {
```

Comment:
We probably cannot directly extend that? I would propose to create a new method, delegate from the old one with DPIUtil.getNativeDeviceZoom() and mark the old one as deprecated. What do you think @HeikoKlare?